lib/pages/main/main_page_bloc.dart

```diff
@@ -110,6 +110,18 @@ class MainPageBloc extends Bloc<MainPageEvent, MainPageState> {
             print("Test");
           }
           break;
+        case MainButtonEvent.refreshPlaylist:
+          var reply = await QuestionDialog.getInstance().show(
+              context: context,
+              question: AppLocalizations.of(context)!
+                  .are_you_sure_you_want_to_refresh_the_playlist);
+          if (reply != DialogQuestionResponse.yes) {
+            return;
+          }
+          if (state.playerState!.isPaused) {
+            await _resumePausePlayer();
+          }
+          await _startRecommendation();
```

Comment:
Jeg kan ikke huske det helt præcis, men var der tale om at tjekke for en null værdi her? Hvis det ikke længere er relevant så ignorer den her kommentar